### PR TITLE
Change %v to %w in errors and other nits

### PR DESCRIPTION
**Pull request by vagababov**
Other are things I noticed when fixing the main %v->%w conversion

### CODE REVIEW

File: testutils/clustermanager/e2e-tests/gke.go [hunk 1]
```diff
@@ -261,11 +261,11 @@ func (gc *GKECluster) checkEnvironment() error {
...
 	output, err = common.StandardExec("gcloud", "config", "get-value", "project")
 	if err != nil {
-		return fmt.Errorf("failed getting gcloud project: '%v'", err)
+		return fmt.Errorf("failed getting gcloud project: %w", err)
 	}
-	if string(output) != "" {
-		project := strings.Trim(strings.TrimSpace(string(output)), "\n\r")
-		gc.Project = project
+	os := string(output)
+	if os != "" {
```
Comment (mattmoor) [inline testutils/clustermanager/e2e-tests/gke.go:267]: you didn't want to fold these together?

Comment (vagababov) [reply testutils/clustermanager/e2e-tests/gke.go]: done.

**Pull request merged**